app/reducers/devices.js

```diff
@@ -202,6 +209,13 @@ const devices = {
     source: {type: 'device', driverId: 'ReliOnPremier'},
     enabled: {mac: true, win: true, linux: true}
   },
+  relionprime: {
+    instructions: 'Make sure the meter is switched off and plug in with cable',
```

Comment:
Should this be wrapped with `i18n.t()`? Also not totally sure about the phrasing